api/admin/controller/__init__.py

```diff
@@ -1740,11 +1740,16 @@ def custom_list(self, list_id):
             # Build the list of affected lanes before modifying the
             # CustomList.
             affected_lanes = Lane.affected_by_customlist(list)
+            for lane in affected_lanes:
```

Comment:
I moved this above the other delete functions because it didn't seem to ever reach this otherwise. I'm not sure if it's the right move.

Reply:
Would there be an easier way to do this via setting a cascade delete constraint in the model?

Reply:
I don't think that's possible for this type of relationship.